src/main/flight/mixer.c

```diff
@@ -623,6 +635,12 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs) {
 #define CRASH_FLIP_DEADBAND 20
 #define CRASH_FLIP_STICK_MINF 0.15f
 
+static float mapThrustToMotorOutput(float thrust, float vbatCompFactor)
+{
+    float linearizedThrust = vbatCompFactor * ((1.0f - thrust_linearization_level) * thrust + thrust_linearization_level * SIGN(thrust) * vbatCompFactor * sqrtf(ABS(thrust)));
```

Comment:
Not sure if the sign management is necessary. I did it because of the 3D mode, but actually I don't know if here we can have a negative thrust value. If not the formula could be cleaned up.

Reply:
ah i didn't notice this, but your completely correct. Thrust should never be able to have a negative sign.

Reply:
Thank you Quick 🙂